src/native/_mutable.js

```diff
@@ -31,6 +31,39 @@ function bufferLastEntry() {
         .concat([str, str.length]);
 }
 
+function callBackLastEntry() {
+  let fn = arguments[arguments.length - 1];
+  if (typeof fn !== 'function') throw Error('A function parameter _must be_ provided')
+
+  let cb = ffi.Callback("void", ['pointer', t.u8Pointer, t.usize],
```

Comment:
you use this - among others -  for the for_each_value. But value has the callback signature `fn(*mut c_void, *const u8,  usize, u64),` - with the last one representing the version number. I think we need to wrap that again into the `{buf: Buffer, version: u64}` we've used other places.